docs/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md

```diff
@@ -54,17 +54,19 @@ The first step is to create the application infrastructure, which includes an ap
 
```

Comment:
Line 53: 1. Open Visual Studio and select **Create a new project** under the **Get started** menu.

This is what people see now in VS 2019: 
![image](https://user-images.githubusercontent.com/12971179/58064508-3b4f9600-7b37-11e9-84e3-50f85d5bad5c.png)